app/models/service_template_transformation_plan_task.rb

```diff
@@ -283,7 +283,7 @@ def conversion_options_source_provider_vmwarews_vddk(_storage)
 
   def conversion_options_source_provider_vmwarews_ssh(storage)
     {
-      :vm_name          => URI::Generic.build(:scheme => 'ssh', :userinfo => 'root', :host => source.host.ipaddress, :path => "/vmfs/volumes").to_s + "/#{storage.name}/#{source.location}",
+      :vm_name          => URI::Generic.build(:scheme => 'ssh', :userinfo => 'root', :host => source.host.ipaddress, :path => "/vmfs/volumes").to_s + "/#{CGI.escape(storage.name)}/#{CGI.escape(source.location)}",
```

Comment:
> .to_s + "/#{CGI.escape(storage.name)}/#{CGI.escape(source.location)}"

This should be part of the path passed to `URI::Generic.build` not tacked on the end